bioimageio/spec/model/v0_3/raw_nodes.py

```diff
@@ -28,6 +28,7 @@
 # same as general RDF
 Badge = Badge
 CiteEntry = CiteEntry
+Dependencies = Dependencies
```

Comment:
Ah, I guess this is a common pattern :D. But it looks weird enough that a comment is warranted.

Reply:
yeah... I don't know what the way to go is there...  just importing it and not using it looks like an unrequired import.

Reply:
yeah, maybe just write a comment to explain it

Reply:
added comments in 44f4d5e